### ISSUE TITLE
Change set X commands to not have set as the first term

### ISSUE THREAD

**Issue by tlmii** [dotnet/HttpRepl#214]
Right now we have two (used to be three until we got rid of `set swagger`) commands that start with `set`:

```console
set base [baseUrl]
set header [headerName] [headerValue]
```
On the other hand we have one command that uses `set` as the second term:

```console
pref set [preferenceName] [preferenceValue]
```
and all other commands begin with something more identifiable to the command itself (`get`, `run`, `ui`, `echo`, 'clear', etc). For instance, it's `echo on` not `set echo on`; it's `cd` not `set dir`

I think we should change the two commands that currently begin with `set` to work differently:

#### `set base [baseUrl]`

This could become one `base` command with two meanings depending on usage:

```console
base
```
would display the current base url

```console
base [baseUrl]
```
would be the equivalent to the existing syntax.

#### `set header [headerName] [headerValue]`

This could similarly have two functions:

```console
header
```
would display the current list of headers

```console
header [headerName] [headerValue]
```
would be the equivalent to the existing syntax.

thoughts? @jodavis @jimmylewis @bhsubra @shanselman

**Comment by shanselman** [comment]
Agreed. Having two "sets" and also having them in different locations is awkward. I'd remove set for regular use and keep it just for pref. Then as you said, header and base are their own thing.

**Comment by tlmii** [comment]
cc @metoly as well.

**Comment by sayedihashimi** [comment]
**Thank you for your interest in HttpRepl.**

This project has been deprecated, and the repository is now archived. As part of this transition, we are closing all open issues.
Please read the https://github.com/dotnet/HttpRepl/issues/701 for alternative recommendations.
